src/app/withdraw/components/withdraw-client.tsx

```diff
@@ -188,7 +197,44 @@ const Withdraw = () => {
       [WalletProvider.XVERSE]: callContractXverse,
       [WalletProvider.LEATHER]: callContractLeather,
       [WalletProvider.FORDEFI]: callContractFordefi,
-      [WalletProvider.ASIGNA]: callContractAsigna,
+      [WalletProvider.ASIGNA]: async ({}: {
+        txHex: string;
+        network: StacksNetworkName | StacksNetwork;
+      }) => {
+        const signedTxHex = await openAsignaContractCall(
```

Comment:
Is `openAsignaContractCall` returning the signed tx, or the txid?

Reply:
tx id